Chirp/Components/Button.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Button.swift
+//  Chirp
+//
+//  Created by Uzoh Okwara on 06.06.23.
+//
+
+import UIKit
+import Lottie
+
+@IBDesignable class Button: UIButton {
+    private let loadingView = LottieAnimationView(name: "button-loader-white")
+    
+    @IBInspectable var isLoading: Bool = false {
+        didSet {
+            loadingView.stop()
+            if isLoading {
+                loadingView.play()
+            }
+            loadingView.isHidden = !isLoading
+            setTitleColor(isLoading ? .clear : .white, for: .normal)
+        }
+    }
+    
+    init() {
+        super.init(frame: .zero)
+        setup()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setup()
+    }
+    
+    private func setup() {
+        addSubview(loadingView)
+        backgroundColor = UIColor(named: "app-brown")
+        clipsToBounds = true
+        layer.cornerRadius = 6
+        
+        contentEdgeInsets = .init(top: 10, left: 10, bottom: 10, right: 10)
```

Comment:
Can we create a constant at the top of the file to hold this value?
```suggestion
        contentEdgeInsets = .init(top: buttonPadding, left: buttonPadding, bottom: buttonPadding, right: buttonPadding)
```